bindings/python/test/objects/test_interval.py

```diff
@@ -396,4 +396,37 @@ def test_real_interval_to_string ():
     assert interval_halfopenleft.to_string() == ']-4.3099999999999996, 3.0]'
     assert interval_halfopenright.to_string() == '[-4.3099999999999996, 3.0['
 
+
```

Comment:
maybe add a `TestInterval` class and put the functions under it? Sorry there is a lot of dirty stuff in there that we are trying to align as we go. Lmk and I can make a pass on it :)

Reply:
👍